rules/rbe_repo.bzl

```diff
@@ -686,6 +700,14 @@ _rbe_autoconfig = repository_rule(
                    "Bazel version. " +
                    "Used internally when use_checked_in_confs is true."),
         ),
+        "copy_resources": attr.bool(
+            default = False,
+            doc = (
+                "Optional. Specifies whether to copy instead of mounting " +
+                "resources such as scripts and project source code to the " +
+                "container for Bazel autoconfig."
```

Comment:
Note that copy is more expensive and should only be enabled where mounting is not enabled